sessionCheckGlobal.js

```diff
@@ -37,12 +37,11 @@
          * Attach a hidden iframe onto the main document body that is used to perform
```

Comment:
Changes should not be made directly in this file; this file is generated from the `npm run build` command. Please make changes to sessionCheck.js and then use that command to update this file.

Reply:
I moved the changes to `sessionCheck.js`, ran `npm run pretest`, fixed the linter violations, ran `npm run build`, then committed all the changes. Did that for each change I made.